cvat/apps/dataset_manager/formats/cvat.py

```diff
@@ -164,7 +187,19 @@ def _parse(cls, path):
                     }
                     subset = el.attrib.get('subset')
             elif ev == 'end':
-                if el.tag == 'attribute' and attributes is not None:
+                if el.tag == 'attribute' and element_attributes is not None and shape_element is not None:
```

Comment:
When is `element_attributes` value can be `not None`? I did not find a place in the code where `element_attributes` is assigned to any value

Reply:
Yes, you're right. Fixed